### PR TITLE
Deleting tests

### PR DESCRIPTION
**Pull request by crmorford**
just wanna throw some more tests in

### CODE REVIEW

**Comment by crmorford** [comment]
and also made cities use just texas, instead of all the zip codes. i think it had too much data.

**Comment by jtlan** [comment]
**cities.js** instead of "**cites.js**" probably...?

File: quicktests/overlaying/tests/realistic/hockey.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+function makeData() {
+  "use strict";  
+
+}
+
+function run(svg, data, Plottable) {
+  "use strict";
+
+  d3.csv("../overlaying/tests/hockey.csv").get(function(error, rows) {
+  var data = rows.reverse();
```
Comment (jtlan) [inline quicktests/overlaying/tests/realistic/hockey.js]: How come you `reverse()`-ed the rows here? Just curious.

File: quicktests/overlaying/tests/realistic/scatter_bar.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  var ds1 = new Plottable.Dataset(data);
+  var ds2 = new Plottable.Dataset(data);
+
+  var bars = new Plottable.Plots.Rectangle()
+    .addDataset(ds1)
+    .x(function (d, i, dataset) { return d.overall; }, xScale)
+    .x2(function(d) { return d.leadership; }, xScale)
+    .y(function(d) { return d.company; }, yScale)
+    .attr("fill", function() { return "#aaaaaa"; })
+    .attr("height", function() { return 20; });
```
Comment (jtlan) [inline quicktests/overlaying/tests/realistic/scatter_bar.js]:
```Javascript
.attr("fill", "#aaaaaa")
.attr("height", 20);
```
should work fine, but it's up to you what style you want to use.

File: quicktests/overlaying/tests/realistic/hockey.js (hunk 1 above)
Comment (crmorford) [inline quicktests/overlaying/tests/realistic/hockey.js]: Ah, i started by copying another example that used csv, and one of you had done this. It worked, so I never changed it, I guess. I'll take it out if it works without.

File: quicktests/overlaying/tests/realistic/scatter_bar.js (hunk 2 above)
Comment (crmorford) [inline quicktests/overlaying/tests/realistic/scatter_bar.js]: ah this was accidental...but should i leave it in to test that?

**Pull request merged**